app/view/sign/signin.html

```diff
@@ -0,0 +1,47 @@
+<%- include sidebar.html %>
+
+<div id='content'>
+  <div class='panel'>
+    <div class='header'>
+      <ul class='breadcrumb'>
+        <li><a href='/'>主页</a><span class='divider'>/</span></li>
+        <li class='active'>登录</li>
+      </ul>
+    </div>
+    <div class='inner'>
+      <% if(typeof(error) !== 'undefined' && error){ %>
```

Comment:
let's remove typeof(), use typeof xxx instead of.

Reply:
done